src/common/explanation/Explanation.tsx

```diff
@@ -7,13 +7,23 @@ type Props = {
   main: string;
   small?: string;
   className?: string;
+  variant?: 'margin' | 'flush';
```

Comment:
It isn't obvious to an outsider what the meaning of flush is, so some kind of comment could be helpful!

Reply:
Added comment.